packages/react-devtools-scheduling-profiler/src/hooks.js

```diff
@@ -7,11 +7,7 @@
  * @flow
  */
 
-import {
-  unstable_createMutableSource as createMutableSource,
-  unstable_useMutableSource as useMutableSource,
-  useLayoutEffect,
-} from 'react';
+import {createMutableSource, useMutableSource, useLayoutEffect} from 'react';
```

Comment:
Are we keeping these prefixed with `unstable_`?

Reply:
Yes. That was the agreement, _but_ DevTools are built with the _experimental_ release channel.